common/djangoapps/util/enterprise_api.py

```diff
@@ -0,0 +1,62 @@
+"""
+Utilities to communicate with the Enterprise service.
```

Comment:
Alrighty, I went ahead and moved it over. I also noticed some preexisting unused imports in `enterprise_helpers` and removed them.